static/manager-dashboard/src/Components/SatisfactionPieChart.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from 'react';
+import { invoke } from '@forge/bridge';
+import PieChart from '../shared/PieChart';
+
+function SatisfactionPieChart() {
+    const [SatisfactionRatingsData, setSatisfactionRatingsData] = useState(null);
+
+    useEffect(() => {
+        invoke('getSatisfactionRatingsData', { example: 'my-invoke-variable' }).then(setSatisfactionRatingsData);
+    }, []);
+
+    const SatisfactionPieChartLabels = ['Bad', 'Somewhat Bad', 'Okay', 'Somewhat Good', 'Good'];
+    const SatisfactionPieChartTitle = 'Satisfaction Ratings';
+
+    return (
+        // Ideally the styling should be in a file, but who likes creating CSS files?
+        <div style={{ margin: "auto", padding: "10px", width: "50%", height: "50%" }}>
```

Comment:
I think it would be nice to add a Performance vs Satisfaction Bar chart in here as well. Can copy the same code in [here](https://github.com/Motive-Metrics/motive-metrics/blob/sharan/performance-ui-update/static/manager-dashboard/src/Components/Performance.js) for the bar chart.